environments/openshift/manifests/observatorium-api-template.yaml

```diff
@@ -0,0 +1,147 @@
+              scheme: HTTP
+            periodSeconds: 30
+          name: observatorium-api
+          ports:
+          - containerPort: 8080
+            name: http
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /-/ready
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 10
+            periodSeconds: 30
+          resources:
+            limits:
+              cpu: ${OBSERVATORIUM_API_CPU_LIMIT}
+              memory: ${OBSERVATORIUM_API_MEMORY_LIMIT}
+            requests:
+              cpu: ${OBSERVATORIUM_API_CPU_REQUEST}
+              memory: ${OBSERVATORIUM_API_MEMORY_REQUEST}
+        - args:
+          - -provider=openshift
+          - -https-address=:8081
+          - -http-address=
+          - -email-domain=*
+          - -upstream=http://localhost:8080
+          - -openshift-service-account=prometheus-telemeter
+          - '-openshift-sar={"resource": "namespaces", "verb": "get", "name": "${NAMESPACE}",
+            "namespace": "${NAMESPACE}"}'
+          - '-openshift-delegate-urls={"/": {"resource": "namespaces", "verb": "get",
+            "name": "${NAMESPACE}", "namespace": "${NAMESPACE}"}}'
+          - -tls-cert=/etc/tls/private/tls.crt
+          - -tls-key=/etc/tls/private/tls.key
+          - -client-secret-file=/var/run/secrets/kubernetes.io/serviceaccount/token
+          - -cookie-secret-file=/etc/proxy/secrets/session_secret
+          - -openshift-ca=/etc/pki/tls/cert.pem
+          - -openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+          - -skip-auth-regex=^/metrics
+          image: ${PROXY_IMAGE}:${PROXY_IMAGE_TAG}
+          name: proxy
+          ports:
+          - containerPort: 8081
+            name: https
+          volumeMounts:
+          - mountPath: /etc/tls/private
+            name: secret-api-tls
+            readOnly: false
+          - mountPath: /etc/proxy/secrets
+            name: secret-api-proxy
+            readOnly: false
+        serviceAccount: prometheus-telemeter
+        serviceAccountName: prometheus-telemeter
+        volumes:
+        - name: secret-api-tls
+          secret:
+            secretName: api-tls
+        - name: secret-api-proxy
+          secret:
+            secretName: api-proxy
+- apiVersion: v1
+  data:
+    session_secret: ""
+  kind: Secret
+  metadata:
+    labels:
+      app.kubernetes.io/name: observatorium-api
+    name: observatorium-proxy
+    namespace: ${NAMESPACE}
+  type: Opaque
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      service.alpha.openshift.io/serving-cert-secret-name: observatorium-tls
+    name: observatorium-api
+    namespace: ${NAMESPACE}
+  spec:
+    ports:
+    - name: http
+      port: 8080
+      targetPort: 8080
+    - name: https
+      port: 8081
+      targetPort: https
+    selector:
+      app.kubernetes.io/name: observatorium-api
+parameters:
+- name: NAMESPACE
+  value: telemeter
```

Comment:
I don't think this namespace exists anywhere, but it will be overwritten anyway.